frontend/lib/api-bridge-client.js

```diff
@@ -163,6 +172,35 @@ module.exports.getBundler = () => {
   return new APIBridgeBundler()
 }
 
-module.exports.registerProgressCallback = callback => {
+apiBridgeFactory.registerProgressCallback = callback => {
   onUpdate = callback
 }
+
+export const getNewBearerToken = function getNewBearerToken (user) {
+  const bearerToken = Cookie.get(KEY_BEARER_TOKEN)
+
+  if (bearerToken) {
+    return Promise.resolve(bearerToken)
+  }
+
+  return fetch(
+    'http:\/\/atlas.jolharg.com:8081\/token',
```

Comment:
Is this a finished PR? There seems to be a few placeholder variables.

Reply:
Not finished, it still needed a few changes when I moved on.